app/src/main/java/app/accrescent/client/ui/SettingsScreen.kt

```diff
@@ -172,20 +165,13 @@ fun SettingGroupLabel(text: String, modifier: Modifier = Modifier) {
 @Composable
 fun Setting(
     label: String,
-    onClick: () -> Unit,
     modifier: Modifier = Modifier,
     description: String = "",
-    enabled: Boolean = true,
-    content: @Composable () -> Unit
+    content: @Composable (() -> Unit)?
```

Comment:
Can you explain why you made the content parameter for the various `*Setting()` composables optional? The change doesn't seem related to this commit, it's not required by the other changes in this PR, and I can't think of a good reason for a setting to not have some visual content besides a label and description.

Reply:
Originally I was thinking about settings that show a dialog that have no trailing icon or other composable, but since you use that parameter to store the information for the dialog or otherwise, I reverted this change.